snapcraft/storeapi/errors.py

```diff
@@ -60,12 +63,21 @@ def __init__(self, extra_information=''):
         super().__init__(message=message)
 
 
-class StoreRetryError(StoreError):
+class StoreNetworkError(StoreError):
 
-    fmt = 'There seems to be a network error: {error}'
+    fmt = 'There seems to be a network error: {message}'
 
     def __init__(self, exception):
-        super().__init__(error=str(exception))
+        message = str(exception)
+        with contextlib.suppress(IndexError):
+            underlying_exception = exception.args[0]
+            if isinstance(underlying_exception,
+                          urllib3.exceptions.MaxRetryError):
+                message = (
+                    'max retries exceeded trying to reach the store\n'
```

Comment:
How about "The maximum amount of retries was exceeded trying to reach the store.\n" - uppercase and period to make it a bit friendler.

Reply:
Most of the extra verbiage there adds no meaning, but I went ahead and used "maximum" as well as ended with a period.